org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/extensions/contentmodel/participants/codeactions/NoGrammarConstraintsCodeAction.java

```diff
@@ -72,7 +72,7 @@ public void doCodeAction(Diagnostic diagnostic, Range range, DOMDocument documen
 					+ document.getTextDocument().lineDelimiter(0);
 			insertText += " xsi:noNamespaceSchemaLocation=\"" + schemaFileName + "\"";
 			CodeAction noNamespaceSchemaLocationAction = createGrammarFileAndBindIt(
-					"Bind to the generated XSD '" + schemaFileName + "' with xsi:noNamespaceSchemaLocation", schemaURI,
+					"Generate '" + schemaFileName + "' and bind with xsi:noNamespaceSchemaLocation", schemaURI,

```

Comment:
I will add DTD/XSD before Generate, because when yo read quickly it's hard to see that it generates a dtd or xsd.

I mean `Generate XSD '.....`

@fbricon what do you think ?

Reply:
I find that redundant, on the contrary

Reply:
Ok thanks for your feedback.